Documentation/Contributing.md

```diff
@@ -295,5 +295,5 @@ Here is an example:
 }
 ```
 
-Once you have add marked your template as `isPreview`, you can see this workbook by adding `feature.includePreviewTemplates` in your Azure Portal Url. So you URL looks something like [https://portal.azure.com/?feature.includePreviewTemplates=true](https://portal.azure.com/?feature.includePreviewTemplates=true)
+Once you have add marked your template as `isPreview`, you can see this workbook by adding `feature.includePreviewTemplates` in your Azure Portal Url. So your URL looks something like [https://portal.azure.com/?feature.includePreviewTemplates=true](https://portal.azure.com/?feature.includePreviewTemplates=true). (Include & between feature flags if also deploying your own gallery or redirecting to a github branch)
```

Comment:
I don think you need to add the part about including &. That is just how you chain query string parameters in any URL

Reply:
Also why is it written [example1](example1). Why is the example written twice, once in brackets and the other in parentheses

Reply:
The URL is written twice so it shows up as a markdown URL, that formatting was already there.